etherpad-wiki.html

```diff
@@ -50,7 +50,7 @@
     var match;
     for (var i = 0; i < lines.length; i++) {
       if (copying) {
-        if (reportStartRegex.exec(lines[i]) || /--+/.exec(lines[i])) {
+        if (reportStartRegex.exec(lines[i]) || /^--+/.exec(lines[i])) {
```

Comment:
We only stop copying if the line starts with `--` (got the issue today, Oleg added `--` in the middle of a text).